modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -396,9 +396,18 @@ spec:
     - role: Contributor
 ----
 
-.. Remove the YAML file for any `CredentialRequest` object that is in Technology Preview or they cause the installation to fail. As of {product-version}, the only credential request in Technology Preview is for the `capi-operator`. To remove this request:
-To remove this request:
-... To list the credential request, run the following command:
+.. Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
++
+[IMPORTANT]
+====
+By default, the release image includes `CredentialsRequest` objects for Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set.
```

Comment:
nit: I'm not sure we need "By default" unless we want to float the possibility of customers building their own release images.

Reply:
Ha! fair point. I'll edit and refresh this :)